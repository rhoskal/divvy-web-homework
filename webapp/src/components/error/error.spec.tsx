import React from "react";
import { render } from "@testing-library/react";

import { Error } from "./Error";

describe("[Component] Error", () => {
  it("should match snapshot", () => {
    const { container } = render(<Error />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
