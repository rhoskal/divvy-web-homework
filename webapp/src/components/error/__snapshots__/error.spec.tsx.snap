// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component] Error should match snapshot 1`] = `
<div>
  <h1>
    Oops!
  </h1>
  <span>
    Well this is embarrassing... ¯\\_(ツ)_/¯
  </span>
</div>
`;
